SwiftLint warnings on RetryIf.swift

```text
⚠️  /Users/jerome/Desktop/nuke/alpha-wallet-ios/modules/AlphaWalletCore/AlphaWalletCore/Extensions/RetryIf.swift:153:47: Vertical Parameter Alignment Violation: Function parameters should be aligned vertically if they're in multiple lines in a declaration. (vertical_parameter_alignment)
/Users/jerome/Desktop/nuke/alpha-wallet-ios/modules/AlphaWalletCore/AlphaWalletCore/Extensions/RetryIf.swift:154:47: warning: Vertical Parameter Alignment Violation: Function parameters should be aligned vertically if they're in multiple lines in a declaration. (vertical_parameter_alignment)
/Users/jerome/Desktop/nuke/alpha-wallet-ios/modules/AlphaWalletCore/AlphaWalletCore/Extensions/RetryIf.swift:155:47: warning: Vertical Parameter Alignment Violation: Function parameters should be aligned vertically if they're in multiple lines in a declaration. (vertical_parameter_alignment)
```

--- a/modules/AlphaWalletCore/AlphaWalletCore/Extensions/RetryIf.swift
+++ b/modules/AlphaWalletCore/AlphaWalletCore/Extensions/RetryIf.swift
@@ -150,10 +150,10 @@ extension Publisher {
        - returns: A publisher that attempts to recreate its subscription to a failed upstream publisher.
        */
     public func retry<S: Combine.Scheduler>(_ resolver: @escaping (Failure) -> RetryBehavior<S>?,
-                                              tolerance: S.SchedulerTimeType.Stride? = nil,
-                                              scheduler: S,
-                                              options: S.SchedulerOptions? = nil) -> AnyPublisher<Output, Failure> {
-
+                                            tolerance: S.SchedulerTimeType.Stride? = nil,
+                                            scheduler: S,
+                                            options: S.SchedulerOptions? = nil) -> AnyPublisher<Output, Failure> {
+        
         return retry(1, resolver: resolver, tolerance: tolerance, scheduler: scheduler, options: options)
     }
 

